Docker/ods-api-db-ods-sandbox/alpine/pgsql/Dockerfile

```diff
@@ -4,8 +4,8 @@
 # See the LICENSE and NOTICES files in the project root for more information.
 
 # Base image with additional packages
-FROM postgres:13.14-alpine3.19@sha256:ea5b664eb639d102308540f11625bc06bbd42e1f386d8b71696a580219ebc6c2 AS base
-RUN apk --no-cache add unzip=~6 dos2unix=~7
+FROM postgres:13.15-alpine3.19@sha256:3f9f1b822982df932fe36b6edee4b0a754b88f6a10949e310cabffc0cfbb6a89 AS base
+RUN apk --upgrade --no-cache add unzip=~6 dos2unix=~7
```

Comment:
Vulnerabilities in this image can be fixed by adding `busybox=~1.36 openssl=~3.1 libxml2=~2.11`